django/db/models/query.py

```diff
@@ -1648,6 +1894,12 @@ def _fetch_all(self):
         if self._prefetch_related_lookups and not self._prefetch_done:
             self._prefetch_related_objects()
 
+    async def _async_fetch_all(self):
+        if self._result_cache is None:
+            self._result_cache = [result async for result in self._iterable_class(self)]
+        if self._prefetch_related_lookups and not self._prefetch_done:
+            sync_to_async(self._prefetch_related_objects)()
+
```

Comment:
What about async variants of remaining methods:
- `values_list()`,
- `values()`,
- `dates()`,
- `datetimes()`,
- `none()`,
- `all()`,
- `filter()`,
- `exclude()`, etc. :thinking:

Reply:
Nvd.